src/tests/functional/shared_test_classes/include/shared_test_classes/base/utils/ranges.hpp

```diff
@@ -117,7 +117,7 @@ static std::map<ov::NodeTypeInfo, std::vector<std::vector<InputGenerateData>>> i
         { ov::op::v1::FloorMod::get_type_info_static(), {{{2, 4}}, {{2, 2, 128}}} },
         { ov::op::v1::Mod::get_type_info_static(), {{{2, 4}}, {{2, 2, 128}}} },
         { ov::op::v1::ReduceMax::get_type_info_static(), {{{0, 5}}, {{-5, 5, 1000}}} },
-        { ov::op::v1::ReduceMean::get_type_info_static(), {{{0, 5}}, {{0, 5, 1000}}} },
+        { ov::op::v1::ReduceMean::get_type_info_static(), {{{0, 5, 1000}}, {{0, 5, 1000}}} },
```

Comment:
@iefode without the change `ReduceOpsLayerTest` with `Mean` op type fails on integral types

Reply:
@iefode `0, 5, 1000` for all types (real and integral) were in original test. With `0, 5` only integral cases are failing in new tests